src/logo/logo.js

```diff
@@ -3,23 +3,23 @@
 import logo from '../models/logo-flat.json';
 import particle from '../textures/particle.png';
 
-import THREE from 'three';
+import * as THREE from 'three';
```

Comment:
Do we need all THREE at once, probably we can import only stuff we need then. Actually it's even better to remove threejs dependency at all (cause it's only used for providing projection matrix). But we should do one sing at a time of course.